packages/app/src/Pages/EventPage.tsx

```diff
@@ -8,5 +8,5 @@ export default function EventPage() {
   const id = parseId(params.id ?? "");
   const thread = useThreadFeed(id);
 
-  return <Thread notes={thread.notes} this={id} />;
+  return <Thread key={id} notes={thread.notes} />;
```

Comment:
Need to remount the component when the note ID changes to reset state of the component.